infrastructure/templates/public-api/components/functionApp.bicep

```diff
@@ -68,67 +95,63 @@ resource appServicePlan 'Microsoft.Web/serverfarms@2023-01-01' = {
   }
 }
 
-var dedicatedStorageAccountName = replace('${subscription}eessa${functionAppName}', '-', '')
+// Configuring a single shared storage account for task management, and 2 individual storage accounts to be split
```

Comment:
Going by the docs, I think the shared storage account is used for normal operations - key management, timer trigger management etc.

I think it would be good to adjust this comment to say that, and that the individual storage accounts are for use by the Durable Functions extension, for reliable execution.

Reply:
I've updated various comments to better reflect this now, thanks!